src/widgetastic_patternfly4/breadcrumb.py

```diff
@@ -0,0 +1,40 @@
+from widgetastic.widget import Widget
+
+
+class BreadCrumb(Widget):
+    """Represents the Patternfly BreadCrumb.
+
+    http://patternfly-react.surge.sh/patternfly-4/components/breadcrumb
+    """
+    ROOT = './/nav[contains(@class, "pf-c-breadcrumb")]/ol'
+    ELEMENTS = './/li'
+
+    def __init__(self, parent, locator=None, logger=None):
+        Widget.__init__(self, parent=parent, logger=logger)
+        self._locator = locator or self.ROOT
+
+    def __locator__(self):
+        return self._locator
+
+    @property
+    def _path_elements(self):
+        return self.browser.elements(self.ELEMENTS, parent=self)
```

Comment:
`parent=self` is not necessary

Reply:
I thought without `parent=self` it will grab all `.//li` elements on the page, while with `parent=self` it will take only those which are in scope of current `ROOT` (meaning, only `li` items of breadcrumb). 
Double checked and can see it's not `Browser.elements()` method (which has default `parent=None`), but `BrowserWrapper.elements()`, and this particular one will pass `parent=self` automatically. This is confusing, but anyhow, thanks for catching.